content/docs/web-components.md

```diff
@@ -20,14 +20,14 @@ class HelloMessage extends React.Component {
 }
```

Comment:
Please translate strings and comments inside codeblocks 😄

Reply:
sorry, which string & comment should be translated?